lib/cube_browser/__init__.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+
+
+class Contourf(object):
+    def __init__(self, cube, coords, **kwargs):
```

Comment:
docstring for the init please, list the args and kwargs

provide links to iris.plot.contourf and matplotlib.pyplot.contourf for kwarg interpretation